TikTokApi/browser.py

```diff
@@ -172,10 +177,9 @@ def sign_url(self, **kwargs):
         else:
             verifyFp = kwargs.get("custom_verifyFp", "verify_khgp4f49_V12d4mRX_MdCO_4Wzt_Ar0k_z4RCQC9pUDpX")
 
-
-        if kwargs.get("custom_did", None) != None:
+        if kwargs.get("custom_did", None) is not None:
```

Comment:
```suggestion
        if kwargs.get("custom_did") is not None:
```
`None` is the default... uh... default for `dict.get()`. If you want `None` to be the result if the key is undefined, you don't have to specify it.